Workbooks/Insights/RedisCache/Insights.workbook

```diff
@@ -156,1042 +156,37 @@
       }
     },
     {
-      "type": 12,
+      "type": 1,
       "content": {
-        "version": "NotebookGroup/1.0",
-        "groupType": "template",
-        "loadFromTemplateId": "community-Workbooks/Common/noSubscriptions",
-        "items": []
+        "json": "This workbook has moved to a new location:"
       },
-      "conditionalVisibility": {
-        "parameterName": "Subscription",
-        "comparison": "isEqualTo",
-        "value": ""
-      },
-      "name": "No Subscriptions group"
+      "name": "text - 1"
     },
     {
-      "type": 12,
+      "type": 11,
       "content": {
-        "version": "NotebookGroup/1.0",
-        "groupType": "editable",
-        "items": [
+        "version": "LinkItem/1.0",
+        "style": "paragraph",
+        "links": [
           {
-            "type": 1,
-            "content": {
-              "json": "{Message}\r\n_Azure Cache for Redis_\r\n<br />\r\n<br />"
-            },
-            "name": "text - 5",
-            "styleSettings": {
-              "margin": "15px 0 10px 10px"
+            "id": "240a0317-bf43-4734-b85b-f33ff1c67a95",
+            "linkTarget": "WorkbookTemplate",
+            "linkLabel": "Open the new version of this workbook",

```

Comment:
As mentioned in the PR description, one small possible concern is that changing this old version of the workbook affects the behavior of the **Insights** > **Azure Cache for Redis** item in the Azure Monitor navigation view:

 ![image](https://user-images.githubusercontent.com/4934473/115167047-f0708d00-a083-11eb-96db-5277754070b0.png)

(At least, that's how it behaves [using the preview feature flag with my changes in a storage account](https://ms.portal.azure.com/?feature.workbookGalleryRedirect=https%3A%2F%2Fbrzagaeus2storage0.blob.core.windows.net%2Fazure-monitor-workbook-templates%2Fpackage#blade/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/redisCacheInsights).)

I probably missed a mention of how the link for **Insights** > **Azure Cache for Redis** gets updated. Will that be updated automatically fairly soon after the changes from this PR are deployed, or would it be better to keep the full original content available in this old woorkbook temporarily until that link can be updated? Thanks again!

Reply:
Now I see how these menu items work, and I realize that the same UI issue will affect the **Insights** item in the Azure Cache for Redis menu too, so for the sake of making the transition as smooth as possible for users, I'll adjust this PR so that it leaves the contents in the old workbook locations unchanged for now. Then, after the menu items in the Azure portal views are updated to point to the new locations, I'll submit another PR to replace the content in the old workbook locations with link steps to the new locations.